pkg/kubectl/deployment.go

```diff
@@ -50,13 +56,21 @@ func (s DeploymentBasicGeneratorV1) Generate(params map[string]interface{}) (run
 	}
 	name, isString := params["name"].(string)
 	if !isString {
-		return nil, fmt.Errorf("expected string, saw %v for 'name'", name)
+		return nil, fmt.Errorf("expected string, saw %#v for 'name'", name)
```

Comment:
It makes the error message more helpful, but yes, it is inconsistent with many of our other error messages. I think the tradeoff is worth it.